content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -107,5 +95,14 @@ Run:
 minikube start
 ```
 
-minikube will automatically select an appropriate driver, but will mention alternatives if available. For more information, see the [minikube drivers documentation](https://minikube.sigs.k8s.io/docs/drivers/).
+minikube will automatically select an appropriate driver, but will mention alternatives if available. For additional information or assistance, please see:
+
+* [minikube documentation](https://minikube.sigs.k8s.io/docs)
+* [#minikube chat on Slack](https://kubernetes.slack.com/archives/C1F5CT6Q1)
+* [minikube issues on GitHub](https://github.com/kubernetes/minikube/issues)
+
+{{% capture whatsnext %}}
 
+* [Running Kubernetes locally via minikube](/docs/setup/learning-environment/minikube/)
```

Comment:
Should we point users to the minikube site? Looks like kind does that too. They just direct you to the kind documentation.

Reply:
@zacharysarah discussed this for a bit, and we came to the shared conclusion that the best user experience was if the introductory tutorial was hosted here, but all of the reference information was linked to the minikube site.